### PR TITLE
React 18

### PR DESCRIPTION
**Pull request by rickhanlonii**
## Overview

This PR adds new docs for React 18.

See the commit list for a summary of all changes.

**Note: this branch is still under development. To add docs to this branch, submit a PR against the `react-18` branch**.

### CODE REVIEW

File: content/blog/2022-03-08-react-18-upgrade-guide.md [hunk 1]
```diff
@@ -248,10 +247,36 @@ In this release, React is dropping support for Internet Explorer, which is [goin
 
 If you need to support Internet Explorer we recommend you stay with React 17.
 
-## Other Changes
+## Deprecations
```
Comment (harish-sethuraman) [inline content/blog/2022-03-08-react-18-upgrade-guide.md:250]: We might want to run `yarn generate-ids` before we could push these changes to the main branch

Comment (harish-sethuraman) [reply content/blog/2022-03-08-react-18-upgrade-guide.md:250]: Never mention only blogs don't seem to have titles

Comment (rickhanlonii) [reply content/blog/2022-03-08-react-18-upgrade-guide.md:250]: Good catch, just ran it and found a few headings we need

File: content/docs/code-splitting.md [hunk 2]
```diff
@@ -81,10 +81,6 @@ When using [Babel](https://babeljs.io/), you'll need to make sure that Babel can
 
 ## `React.lazy` {#reactlazy}
 
-> Note:
->
-> `React.lazy` and Suspense are not yet available for server-side rendering. If you want to do code-splitting in a server rendered app, we recommend [Loadable Components](https://github.com/gregberge/loadable-components). It has a nice [guide for bundle splitting with server-side rendering](https://loadable-components.com/docs/server-side-rendering/).
-
```
Comment (shobhitsharma) [inline content/docs/code-splitting.md:87]: This change.

File: content/versions.yml [hunk 3]
```diff
@@ -23,42 +26,22 @@
 - title: '16.9'
   changelog: https://github.com/facebook/react/blob/main/CHANGELOG.md#1690-august-8-2019
 - title: '16.8'
-  path: /version/16.8
   changelog: https://github.com/facebook/react/blob/main/CHANGELOG.md#1680-february-6-2019
-  url: https://5d4b5feba32acd0008d0df98--reactjs.netlify.com/
```
Comment (slorber) [inline content/versions.yml:28]: @acdlite curious to know why you removed these Netlify permalinks.

Did they go offline without notice? I'm facing this problem on the Docusaurus versioning page: https://github.com/facebook/docusaurus/issues/8634

**Pull request merged**